GVFS/GVFS.Common/Git/GitObjects.cs

```diff
@@ -155,33 +155,13 @@ public bool TryWriteMultiPackIndex(ITracer tracer, GVFSEnlistment enlistment, Ph
             using (ITracer activity = tracer.StartActivity(nameof(this.TryWriteMultiPackIndex), EventLevel.Informational, Keywords.Telemetry, metadata: null))
             {
                 GitProcess process = new GitProcess(enlistment);
-                GitProcess.Result result = process.WriteMultiPackIndex(enlistment.GitPackRoot);
+                GitProcess.Result result = process.WriteMultiPackIndex(enlistment.GitObjectsRoot);
 
-                if (!result.HasErrors)

```

Comment:
Are we going to leave any midx files in the pack directory and the user will have to clean up if they so desire?  Also I'm assuming git is what will keep the midx in the objects directory cleaned up?

Reply:
The new mechanism is to have a constant-name file ($objdir/pack/multi-pack-index) that is swapped out like the .git/index file (and the $objdir/info/commit-graph file).

We will probably want to delete the old MIDX files at some point. I wonder if that clean up should be an upgrade step, or if we should leave them in-place for a version so users can downgrade and still use their old midxes if they want.

Reply:
The old code uses a `midx-head` file containg a hash that points to a `midx-<hash>.midx` file. The new code only knows about the `multi-pack-index` file.

So, when a user downgrades, they will still have their old `midx-head` as a valid pointer, and the next prefetch would extend it.

We can do a cleanup step after this multi-pack-index version is released.

Reply:
Ok thanks!